src/lib/storage/fixed_width_integer_vector.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "abstract_attribute_vector.hpp"
+#include "abstract_segment.hpp"
+#include "types.hpp"
+#include "vector"
+
+namespace opossum {
+
+template <typename T>
+class FixedWidthIntegerVector : public AbstractAttributeVector {
+ public:
+  explicit FixedWidthIntegerVector(size_t size);
+
+  ValueID get(const size_t index) const;
```

Comment:
For virtual methods, it is recommended to specify one of `virtual`, `override` or `final`, see http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-override